controllers/page/PageHandler.inc.php

```diff
@@ -164,7 +164,7 @@ private function setupHeader($args, $request) {
 			$dispatcher = $request->getDispatcher();
 			$contextsNameAndUrl = array();
 			while ($workingContext = $workingContexts->next()) {
-				$contextUrl = $dispatcher->url($request, ROUTE_PAGE, $workingContext->getPath());
+				$contextUrl = $dispatcher->url($request, ROUTE_PAGE, $workingContext->getPath(), 'dashboard');
```

Comment:
This is OK in OMP, but beware that it'll affect the journal switcher in the OJS front-end; not all users will have access to the dashboard, and sending them there via the front-end journal switcher will be confusing.